README.md

```diff
@@ -1,50 +1,76 @@
+Minor changes that do not broadly impact the project's functionality or user experience may not require an RFC. This includes:
 
-1. **Participate in the Discussion**: Engage with the community and project maintainers in the PR discussion. This is where your proposal will be reviewed, and feedback will be provided.
+- Bug fixes
+- Minor feature changes that do not alter key functionality
+- Minor UI/UX adjustments that do not alter the overall design
+- Documentation updates
 
-1. **Finalizing the RFC**: The discussion period will remain open until the RFC is deemed complete. Upon completion, the PR will be closed. Note that closing the PR does not necessarily mean the RFC is approved; it signifies the end of the discussion phase.
+## Timeline
 
-## Community Participation
+The typical timeline for an RFC process, which is a minimum of 3 weeks, is as follows:
 
-We highly encourage community involvement in the RFC process. Whether you're proposing an RFC or contributing to the discussion of someone else's, your input is valuable.
+1. **Proposal Submission**: The RFC is submitted to the repository and made available for public comment.
+1. **Discussion Period**: A minimum of 2 weeks is allocated for community discussion and feedback. This period may be extended based on the complexity of the RFC or the volume of feedback received.
+1. **Revision Period**: (optional) Up to 1 week for the proposer to incorporate feedback and make necessary revisions to the RFC.
+1. **Final Decision**: Within 1 week after the revision period or discussion period, a decision will be made regarding the RFC's implementation.
 
-- **Informal Discussions**: Before submitting an RFC, you may want to discuss your idea informally. Join us on our [Discord](https://discord.com/invite/ktMAKGBnBs) for preliminary discussions and feedback from the community.
+## Decision Making
+### Handling Open Questions
 
-## Contribution Guidelines
+If open questions remain as the discussion timeline concludes:
 
-While contributing to the Meshtastic RFC repository, please adhere to our community standards and guidelines. These are in place to ensure a productive and respectful environment for collaboration.
+- The RFC proposer, in collaboration with core contributors, will assess whether additional information or revisions are needed to address these questions.
+- If consensus cannot be reached within the allotted time, the decision will be escalated to the Meshtastic administrators for review.
 
-- Be respectful and constructive in your communication.
-- Clearly articulate your proposals and feedback.
-- Engage in discussions with an open mind.
+### Approval Process
+
+For an RFC to be merged and accepted:
+
+1. **Consensus Among Core Contributors**: There must be approval by _at least_ one core contributor from each project/repository that will be impacted by the RFC during the 1 week **Final Decision** period.
+  - **Core Contributor**: A core contributor is someone who
```

Comment:
Okay I have no idea here. Is core contributor even the right word? Maintainer? Something else? We need to define what/who they are.

Reply:
I like maintainer, because to me a maintainer feels like it conveys more of a sense of stewardship of the project.

Reply:
I like that. How do we define who that is, objectively?